schwifty/bank_registry/generated_be.json

```diff
@@ -6270,5 +6270,21 @@
     "bank_code": "984",
     "name": "bpost",
     "short_name": "bpost"
+  },
+  {
+    "country_code": "BE",
+    "primary": true,
+    "bic": "BPOTBEBE",
+    "bank_code": "BPOT",
```

Comment:
The domestic bank code for Belgian banks is numeric https://wise.com/gb/iban/belgium

Reply:
@mdomke is [this](https://wise.com/gb/swift-codes/BPOTBEBEXXX) not valid? 
meaning that the bank code resolves to `BPOT`?
if not, how can one go and find the appropriate numeric bank code? I've looked for it but haven't been able to find it

Reply:
The BIC has a bank code segment, which in case of `BPOTBEBE` is indeed `BPOT`, since the genral format is
```
BBBBccLLbbb
B = bank code
c = country code
L = location code
b = branch code
```
The bank code in this scope is however the domestic bank code which is used as part of the IBA, since the bank code is required to link the IBAN and BIC objects. This is actually the hard part about the whole information gathering around the bank registries. Sometimes this information is provided by the national central bank. T